docs/changelog/97405.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 97405
```

Comment:
When labelling this as a non issue, this changelog can be removed.

Reply:
When I changed the label, the changelog was removed, and a rebuild did not happen - cool!

Reply:
Spoke too soon. Rebuild running. I'll wait for that to finish before merging.